recipes/baybe/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "baybe" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/emdgroup/baybe/archive/{{ version }}.tar.gz
+  sha256: 69ce8a86b5c6c4c4208cfba1159775445ed1386a685acd79dfa964168ea3433b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<3.12
+    - setuptools >=45
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >=3.8,<3.12
+    - attrs >=22.2.0
+    - botorch >=0.8.1
+    - cattrs >=23.2.0
+    - funcy >=1.17
+    - gpytorch >=1.9.1
+    - ngboost >=0.3.12
+    - numpy >=1.24.1
+    - pandas >=1.4.2
+    - protobuf <=3.20.3
+    - scikit-learn >=1.1.1
+    - scikit-learn-extra >=0.3.0
+    - scipy >=1.10.1
+    - setuptools-scm >=7.1.0
+    - pytorch >=1.11.0
+    - baybe
```

Comment:
The package cannot depend on itself.

```suggestion
```

Reply:
Noticing missing conda deps in my recipe. I'm on it...